triage/render.js

```diff
@@ -128,66 +172,75 @@ function renderCluster(top, key, keyId, text, tests) {
     return count == 1 ? count + ' ' + word : count + ' ' + word + suffix;
   }
 
+  var counts = clustered.makeCounts(keyId);
+
   var clusterSum = clustersSum(tests);
   var recentCount = clustered.getHitsInLastDayById(keyId);
   var failureNode = addElement(top, 'div', {id: keyId}, [
-    createElement('h2',
-      {innerHTML: `${plural(clusterSum, 'FAILURE', 'S')} (${recentCount} RECENT) MATCHING <a href="#${keyId}" class="key">${keyId}</a>`}),
+    createElement('h2', null, [
+      `${plural(clusterSum, 'FAILURE', 'S')} (${recentCount} RECENT) MATCHING `,
+      createElement('a', {href: '#' + keyId}, keyId),
+      createElement('a', {href: 'https://github.com/search?type=Issues&q=' + keyId, target: '_blank', rel: 'noopener'}, 'github search')
+    ]),
     createElement('pre', null, options.showNormalize ? key : text),
     createElement('div', {className: 'graph', dataset: {cluster: keyId}}),
   ]);
-  var clusterJobs = addElement(failureNode, 'ul');
-  var list = addElement(failureNode, 'ul');
+  var list = addElement(failureNode, 'ul', null, [
+    createElement('li', {innerText: `Recent Failures ${rightArrow}`}, [
```

Comment:
Please make this a superlative so it is more clear (newest failures, most recent failures).

Also try and use a different word. Why does the H2 say 1316 RECENT FAILURES and yet when I expand recent failures there are only 5.